resources-manager/include/ISystemResourceEntity.hpp

```diff
@@ -14,18 +14,41 @@
 
 #include <cstdint>
 #include <string>
+#include <vector>
+#include "utilization.hpp"
 
 namespace concord::performance {
 class ISystemResourceEntity {
  public:
+  // ps - per second
```

Comment:
Implementations of ISystemResourceEntity should cover these scenarios and return measurements.